config/database.yml

```diff
@@ -2,6 +2,10 @@ defaults: &defaults
   adapter: mysql2
   encoding: utf8mb4
   pool: 16
+  database: <%= ENV["MYSQL_DATABASE"] %>
+  username: <%= ENV["MYSQL_USER"] %>
+  password: <%= ENV["MYSQL_PASSWORD"] %>
+  host: <%= ENV["DATABASE_URL"] || '127.0.0.1' %>
```

Comment:
It is odd that the previous code did not work for you. Maybe it is a Mac thing? We can leave it as is for now.

Reply:
I'll skip this file in the future when pushing to remote repository. It should be just a configuration thing so let's use your version in the integration branch.